### PR TITLE
Created TAC Minutes & Supporting Docs For 2023-11-30 Meeting

### PR DESCRIPTION
**Pull request by Riaankl**
@bensternthal can you please review?

### CODE REVIEW

File: TAC/Meetings/2023/2023-11-30/TAC Minutes 2023-11-30.md
Comment (lkatalin) [inline TAC/Meetings/2023/2023-11-30/TAC Minutes 2023-11-30.md:54]:
```suggestion
* Red Hat - Lily Sturmann  + / Yash Mankad +
```

Comment (Riaankl) [reply TAC/Meetings/2023/2023-11-30/TAC Minutes 2023-11-30.md]: Thank you of point this out. Updated.

File: TAC/Meetings/2023/2023-11-30/TAC Minutes 2023-11-30.md
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-11-30/TAC Minutes 2023-11-30.md:67]:
```suggestion
* Hani Benhabiles
```

**Review by dcmiddle — APPROVED** [review]

**Review by AlecFernandez — APPROVED** [review]

**Review by AlecFernandez — APPROVED** [review]

**Review by yashkmankad — APPROVED** [review]

**Comment by Riaankl** [comment]
Great! We have 4 approvals. Who has merge permission?

**Comment by dcmiddle** [comment]
> Great! We have 4 approvals. Who has merge permission?

Great job ushering this through and thanks for everyone engaging asynchronously. I'll show the minutes briefly in the TAC meeting and then I can merge.

**Pull request merged**
